3rdparty/python/constraints.txt

```diff
@@ -1,38 +1,3 @@
-# Generated by build-support/bin/generate_lockfile.sh on Tue Nov 24 23:29:42 MST 2020
-ansicolors==1.1.8
-attrs==20.3.0
-beautifulsoup4==4.6.3
-certifi==2020.11.8
-cffi==1.14.4
-chardet==3.0.4
-cryptography==3.2.1
-fasteners==0.15
-idna==2.10
-importlib-metadata==3.1.0
-iniconfig==1.1.1
-monotonic==1.5
-more-itertools==8.6.0
-mypy==0.782
-mypy-extensions==0.4.3
-packaging==20.4
-pathspec==0.8.0
-pex==2.1.21
-pip==20.2.1
-pluggy==0.13.1
-psutil==5.7.0
-py==1.9.0
-pycparser==2.20
-pyOpenSSL==19.1.0
-pyparsing==2.4.7
-pystache==0.5.4
-pytest==6.0.2
-PyYAML==5.3.1
-requests==2.25.0
-setproctitle==1.2
-setuptools==50.3.2
-six==1.15.0
-toml==0.10.2
-typed-ast==1.4.1
-typing-extensions==3.7.4.2
-urllib3==1.26.2
-zipp==3.4.0
+# Generated by build-support/bin/generate_lockfile.sh on Thu Dec 10 16:59:34 PST 2020
```

Comment:
This list should be much bigger. Perhaps something is borked with `build-support/bin/generate_lockfile.sh`.

Reply:
will run again.